### DISCUSSION TITLE
What are the differences (in application) between mev-flood and the transaction spammer?

### DISCUSSION THREAD

**Discussion by leeederek** [ethpandaops/ethereum-package#123] [Q&A] [answered]
I did some reading and it looks like `transaction_spammer` gets spun up regardless of whether the MEV infrastructure gets spun up ([source](https://github.com/kurtosis-tech/eth2-package/blob/gyani/full-mev/main.star#L99)), so I'm curious:
1. Do the transactions from the transaction_spammer AND mev-flood both get sent to the MEV-builder or mock-builder to create bundles for the relayer to forward to each node?
2. [Why is mev-flood only spun up for mock-mev](https://github.com/kurtosis-tech/eth2-package/blob/gyani/full-mev/main.star#L71)? 
3. I did some reading on mev-flood: it first deploys uniswap v2 liquidity pool smart contracts, funds those pools, and then sends a series of continuous uni v2 swaps to the network its connected to. Does mev-flood do anything else in the eth2-package?

**Comment by lfluse** [comment]
Thx. It's realy

**Comment by Soulchild07** [comment]
thx

**Comment by leeederek** [comment, marked as answer]
Did some more research, adding my findings here:
1. Yes
2. MEV-Flood gets spun up for full-mev and mock-mev to simulate MEV activity 
3. No

**Comment by MrChemodan** [comment]
good job

**Comment by WhiteZip57** [comment]
great job
